server/build.gradle

```diff
@@ -112,6 +112,7 @@ dependencies {
     Classpaths.inheritImmutables(project, true)
 
     runtimeOnly project(':authentication:example-providers:psk')
+    runtimeOnly project(':plugin-trackedseekablechannel')
```

Comment:
I don't think we need this here, assuming we have the runtimeOnly dependency in extensions-parquet-table.